students/chan/westagram/users/views.py

```diff
@@ -42,9 +48,12 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
-            if not User.objects.filter(email=email, password=password).exists():
+            if not User.objects.filter(email=email).exists():
                 return JsonResponse({'Message' : 'INVALID_USER'}, status=401)
 
+            if not bcrypt.checkpw(password.encode('utf-8'),User.objects.filter(email=email).get().password.encode('utf-8')):
```

Comment:
로직은 맞으나 가독성을 조금 더 좋게 하기 위해 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
            user = User.objects.get(email=email)
            
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
```